src/Symfony/Component/Serializer/Tests/SerializerTest.php

```diff
@@ -1024,15 +1024,6 @@ public function testCollectDenormalizationErrors(?ClassMetadataFactory $classMet
                 'useMessageForUser' => true,
                 'message' => 'Failed to create object because the class misses the "constructorArgument" property.',
             ],
-            [
-                'currentType' => 'string',
-                'expectedTypes' => [
-                    'float',
-                ],
-                'path' => 'php74FullWithTypedConstructor',
-                'useMessageForUser' => false,
-                'message' => 'The type of the "something" attribute for class "Symfony\Component\Serializer\Tests\Fixtures\Php74FullWithTypedConstructor" must be one of "float" ("string" given).',
-            ],
```

Comment:
Is this change needed?

Reply:
Unfortunately yes, but it was kind of a duplicate as there is already a more precise exception about the `something` parameter

Reply:
Ah I see, this is because we group errors by path and will already have an entry for `php74FullWithTypedConstructor.something` right below.